### PR TITLE
test common::utils::timestamp

### PR DESCRIPTION
**Pull request by 0xgleb**
## Motivation

- `common::utils::timestamp` is missing tests

## Solution

- add unit tests

## Checks

By submitting this for review, I'm confirming I've done the following:
- [X] made this PR as small as possible
- [X] unit-tested any new functionality
- [X] linked any relevant issues or PRs
- [X] included screenshots (if this involves a front-end change)

## Summary by CodeRabbit

- **Tests**
  - Added new unit tests to verify accurate timestamp formatting across different time zones and edge cases.
- **Refactor**
  - Enhanced error handling to support equality checks for better test validation.
- **Style**
  - Improved code readability by removing a redundant blank line.

### CODE REVIEW

File: crates/common/src/utils/timestamp.rs [hunk 1]
```diff
@@ -29,3 +30,41 @@ pub fn format_timestamp_display(timestamp: i64) -> Result<String, FormatTimestam
...
+    let earliest_valid = "86400".to_string(); // 1 day after epoch
+    let result = format_bigint_timestamp_display(earliest_valid.clone());
+    assert!(result.is_ok());
+
+    // Test latest reasonably valid timestamp
+    // December 31, 2099 (arbitrary future date that should be valid)
+    let future_valid = "4102444800".to_string();
+    let result = format_bigint_timestamp_display(future_valid.clone());
+    assert!(result.is_ok());
+}
```
[the remark this answers is not in the document]
Comment (0xgleb) [reply crates/common/src/utils/timestamp.rs:70]: there are tests for error cases and concurrently changing env vars can lead to unpredictable behavior

**Review by findolor — APPROVED** [review]

**Review by hardyjosh — APPROVED** [review]
